CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 # 5.15.4 / 4.15.4
 
 -   Fix process.env replacement in build [#2267](https://github.com/mobxjs/mobx/pull/2267) by [@fredyc](https://github.com/fredyc)
+-   Add error when computed value declared for unspecified getter [#1867](https://github.com/mobxjs/mobx/issues/1867) by [@berrunder](https://github.com/berrunder)  
```

Comment:
Please move this above this version which is already published. I wonder you haven't noticed the text on the first line?

Reply:
Fixed

Reply:
Thanks. Can you help me out a little what made you decide to add that line to a released version? This happens to most people so I am trying to figure out how to make it more apparent it should go to a different place.

Reply:
Honestly, i was just looking for examples in open PRs and found change in #2312, where line also added under already published version. And I was tired in the evening - it helped to miss first line too, i guess.